docs/api.rst

```diff
@@ -3114,35 +3114,45 @@ Get asset identifiers mappings
       Content-Type: application/json
 
       {
-          "result": {
-              "eip155:1/erc20:0xB6eD7644C69416d67B522e20bC294A9a9B405B31": {
-                  "name": "0xBitcoin",
-                  "symbol": "0xBTC",
-                  "asset_type": "evm token"
+        "result":{
+            "assets":{
+              "eip155:1/erc20:0xB6eD7644C69416d67B522e20bC294A9a9B405B31":{
```

Comment:
why remove the space between `:` and `{` ? This is how json is styled in this file . Same in the rest of the diff